### PR TITLE
Adjust timeout for create and get operations

### LINKED ISSUE #1886: Adjust timeout for create and get operations
## Issue Details

Current timeout are very short in certain situations.
Examples:

Security keys: As these devices are external and depends on user behavior like locate/retrieve/use/put it back, often it leads to timeout of the operation and leads to bad user experience.
Phones: With Hybrid testing, while scanning QR codes, user has been found in similar situations as above described for security keys.

## Proposed Change
Adjust timeout range to 5-10 mins. Default timeout: 5 mins.

### PR DESCRIPTION
**Pull request by akshayku**
Current timeout are very short in certain situations. 
Examples:
- Security keys: As these devices are external and depends on user behavior like locate/retrieve/use/put it back, often it leads to timeout of the operation and leads to bad user experience. 
- Phones: With Hybrid testing, while scanning QR codes, user has been found in similar situations as above described for security keys. 

Proposal:
Adjust timeout range to 5-10 mins. Default timeout: 5 mins.

Fixes #1886

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/1885.html" title="Last updated on May 3, 2023, 12:32 PM UTC (6500d31)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/1885/721f151...6500d31.html" title="Last updated on May 3, 2023, 12:32 PM UTC (6500d31)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1747,15 +1747,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
...
-                :   is set to {{UserVerificationRequirement/discouraged}}
-                ::  Recommended range: 30000 milliseconds to 180000 milliseconds.
-                ::  Recommended default value: 120000 milliseconds (2 minutes).
-
-                :   is set to {{UserVerificationRequirement/required}} or {{UserVerificationRequirement/preferred}}
-                ::  Recommended range: 30000 milliseconds to 600000 milliseconds.
-                ::  Recommended default value: 300000 milliseconds (5 minutes).
+                :  Recommended range: 300000 milliseconds to 600000 milliseconds.
+                :  Recommended default value: 300000 milliseconds (5 minutes).
             </dl>
```
Comment (emlun) [inline index.bs:1753]: The `switch` construct doesn't seem quite appropriate for this. I suggest replacing the `<dl>`s with the bulleted list syntax like below, around https://github.com/w3c/webauthn/pull/1885/files#diff-5e793325cd2bfc452e268a4aa2f02b4024dd9584bd1db3c2595f61f1ecf7b985R1801-R1803 .

File: index.bs [hunk 2]
```diff
@@ -2255,15 +2249,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
...
-                    :   is set to {{UserVerificationRequirement/discouraged}}
-                    ::  Recommended range: 30000 milliseconds to 180000 milliseconds.
-                    ::  Recommended default value: 120000 milliseconds (2 minutes).
-
-                    :   is set to {{UserVerificationRequirement/required}} or {{UserVerificationRequirement/preferred}}
-                    ::  Recommended range: 30000 milliseconds to 600000 milliseconds.
-                    ::  Recommended default value: 300000 milliseconds (5 minutes).
+                    :  Recommended range: 300000 milliseconds to 600000 milliseconds.
+                    :  Recommended default value: 300000 milliseconds (5 minutes).
                 </dl>
```
Comment (emlun) [inline index.bs:2255]: The `switch` construct doesn't seem quite appropriate for this. I suggest replacing the `<dl>`s with the bulleted list syntax like at https://github.com/w3c/webauthn/pull/1885/files#diff-5e793325cd2bfc452e268a4aa2f02b4024dd9584bd1db3c2595f61f1ecf7b985R1801-R1803 .

**Review by emlun — CHANGES_REQUESTED** [review]

**Review by timcappalli — APPROVED** [review]
lgtm

**Review by agl — APPROVED** [review]

**Review by ve7jtb — APPROVED** [review]
lgtm

**Pull request merged**